azurepipelines/e2e_test/packer/deltagen/README.md

```diff
@@ -0,0 +1,50 @@
+# DeviceUpdate Docker Image to Generate Microsoft Delta Update SWUpdate Diff Payload
+
+## Overview
+
+The hcl2 file in this dir for easily creating and maintaining reproducible
+builds of the docker image that is used in the e2e test pipeline to generate
+delta update payload. The tool used is
+[Hashicorp Packer](https://developer.hashicorp.com/packer).
+
+The scripts run by packer on the base container to set up the container image
+are located in the `scripts/` directory (excluding run.sh)
```

Comment:
run.sh is not run by the packer scripts for building the base container, but is for invoking the container after having done `docker run` to generate the delta artifact. It may be better to have it in a different dir to distinguish between scripts run for building the docker image via packer vs. script used for executing the container, but leaving as-is for now.